docs/ops/pooling/MaxPool_1.md

```diff
@@ -6,7 +6,79 @@
 
 **Short description**: [Reference](http://caffe.berkeleyvision.org/tutorial/layers/pooling.html)
 
-**Detailed description**: [Reference](http://cs231n.github.io/convolutional-networks/#pool)
+**Detailed description**: [Reference](https://deeplizard.com/learn/video/ZjM_XQa5s6s). Input shape can be either 3D, 4D or 5D. Pooling operation is performed with the respect to input shape from the third dimension to the last dimension. If paddings are used then during the pooling calculation their value is `-inf`.  
+        `H_out = (H + pads_begin[0] + pads_end[0] - kernel[0] / strides[0]) + 1`  
+        `W_out = (H + pads_begin[1] + pads_end[1] - kernel[1] / strides[1]) + 1`  
+        `D_out = (H + pads_begin[2] + pads_end[2] - kernel[2] / strides[2]) + 1`  
+
+Example 1 shows how *MaxPool* operates with `auto_pad = explicit`
+
+```
```

Comment:
Please move these examples before IR examples. Because at this moment it does not mention about padding mode attribute and its description.